### PR TITLE
Adding jax backend

### PR DESCRIPTION
**Pull request by mrava87**
WIP... moved to https://github.com/PyLops/pylops/pull/590

### CODE REVIEW

File: pylops/jaxoperator.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+from pylops import LinearOperator
+
+
+class JaxOperator(LinearOperator):
+    def __init__(self, Op):
+        super().__init__(dtype=Op.dtype, dims=Op.dims, dimsd=Op.dimsd, name=Op.name)
+        self._matvec = jax.jit(Op._matvec)
+        self._rmatvec = jax.jit(Op._rmatvec)
+
+    def _rmatvecad(self, x, y):
```
Comment (VascoSch92) [inline pylops/jaxoperator.py:17]: types are missing in signature

File: pylops/jaxoperator.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+
+import jax
+
+from pylops import LinearOperator
+
+
+class JaxOperator(LinearOperator):
+    def __init__(self, Op):
+        super().__init__(dtype=Op.dtype, dims=Op.dims, dimsd=Op.dimsd, name=Op.name)
```
Comment (VascoSch92) [inline pylops/jaxoperator.py:13]: if you put a comma at the end it will be more readable

**Comment by mrava87** [comment]
@VascoSch92 thanks for the comments! This is a very early days draft so we will clean things up later when the full structure is in place :)

**Comment by VascoSch92** [comment]
> @VascoSch92 thanks for the comments! This is a very early days draft so we will clean things up later when the full structure is in place :)

No problem :-) I will wait for the final version ;-)

File: pylops/jaxoperator.py (hunk 1 above)
Comment (mrava87) [reply pylops/jaxoperator.py]: Done

File: pylops/jaxoperator.py (hunk 2 above)
Comment (mrava87) [reply pylops/jaxoperator.py:23]: not sure what you mean...

**Comment by mrava87** [comment]
@VascoSch92 see https://github.com/PyLops/pylops/pull/590 if still interested to review with PR :) I will soon close this one...

**Comment by VascoSch92** [comment]
I will give a look tomorrow :)

**Pull request closed without merging**